common/io/Serial.cpp

```diff
@@ -231,9 +268,11 @@ bool AcquireUUCPLockAndOpen(const std::string &path, int oflag, int *fd) {
 #endif  // HAVE_SYS_IOCTL_H
   return true;
 }
+#endif // UUCP_LOCKING
 
 
 void ReleaseUUCPLock(const std::string &path) {
+#ifdef UUCP_LOCKING
```

Comment:
I think it would be good to be consistent here too and create a ReleaseLock function, where the non UUCP path does nothing.

Reply:
Done: added ReleaseSerialPortLock